### DISCUSSION TITLE
Instação

### DISCUSSION THREAD

**Discussion by joaopaulospp** [chatwoot/chatwoot#4850] [Chatwoot APIs]
Como faz a instalação ??

**Comment by converso-chat** [comment]
Bom Dia João Paulo. Se precisar te ajudo a instalar, ***@***.***

Em seg., 13 de jun. de 2022 às 09:08, joaopaulospp ***@***.***>
escreveu:

**Comment by sojan-official** [comment]
https://www.chatwoot.com/docs/self-hosted
